native/src/modules/categories/components/Categories.js

```diff
@@ -50,29 +49,25 @@ type PropsType = {|
  */
 class Categories extends React.Component<PropsType> {
   onTilePress = (tile: TileModel) => {
-    const { cityCode, language, navigateToCategory } = this.props
-    navigateToCategory({ cityCode, language, path: tile.path })
+    const { cityModel, language, navigateToCategory } = this.props
+    navigateToCategory({ cityCode: cityModel.code, language, path: tile.path })
   }
 
   onItemPress = (category: { title: string, thumbnail: string, path: string }) => {
-    const { cityCode, language, navigateToCategory } = this.props
-    navigateToCategory({ cityCode, language, path: category.path })
+    const { cityModel, language, navigateToCategory } = this.props
+    navigateToCategory({ cityCode: cityModel.code, language, path: category.path })
```

Comment:
Somehow I get an error here, when e.g. navigate to Testumgebung -> Willkommen -> Willkokommen in [Stadt]. It seems that the cityModel is not available here...

Reply:
Should be fixed, could you check again please?